src/shared/dissect-image.c

```diff
@@ -355,7 +355,8 @@ int dissect_image(int fd, const void *root_hash, size_t root_hash_size, DissectI
                 /* Filter out weird MMC RPMB partitions, which cannot reasonably be read, see
                  * https://github.com/systemd/systemd/issues/5806 */
                 sysname = udev_device_get_sysname(q);
-                if (sysname && startswith(sysname, "mmcblk") && endswith(sysname, "rpmb"))
+                if (sysname && startswith(sysname, "mmcblk") &&
+                        (endswith(sysname, "rpmb") || endswith(sysname, "boot0" ) || endswith(sysname, "boot1")))
```

Comment:
are there exactly two of these called "boot0" and "boot1"? Or could there be "boot2"?

I wished we wouldn't have to special case things like this here...

Reply:
I've only found this on side 3:
https://www.slideshare.net/linaroorg/intro-to-emmc
eMMC 4.3 specify only 2 boot partitions (for bootloader redundancy)
https://www.jedec.org (registration needed) would be the original source of the specification.